cirq-web/cirq_ts/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "web",
+  "version": "0.12.0",
+  "description": "Web widgets for Cirq",
+  "main": "index.js",
+  "private": true,
+  "dependencies": {
+    "gts": "^3.1.0",
+    "live-server": "^1.2.1",
+    "tsc-watch": "^4.2.9",
+    "typescript": "^4.2.4"
+  },
+  "devDependencies": {
+    "@types/three": "^0.128.0",
+    "concurrently": "^6.1.0",
+    "three": "^0.128.0",
+    "ts-loader": "^9.1.2",
+    "webpack": "^5.37.0",
+    "webpack-cli": "^4.7.0",
+    "webpack-dev-server": "^4.0.0-beta.0",
+    "gts": "^3.1.0",
+    "typescript": "^4.0.3",
+    "@types/node": "^14.11.2"
+  },
+  "scripts": {
+    "start": "webpack serve --mode development",
+    "build": "npm test && webpack --mode production",
+    "test": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha -r ts-node/register 'test/**/*.ts'",
+    "lint": "gts lint",
```

Comment:
It may be the lint configuration or maybe some manual error. I ran it a few times and it auto fixed files `npm run fix`. I had it down to where there were no errors except one complaining that three.js was an unpublished package (I think the rule was `no-unpublished-package`?). Going to look into that later though and also maybe check the config to make sure everything is in line.